### ISSUE TITLE
Add custom loss function

### ISSUE THREAD

**Issue by ACALJJ32** [open-mmlab/mmediting#486]
How do I implement a new loss function in mmedit?

**Comment by ckkelvinchan** [comment]
1. Implement the loss in `mmedit/models/losses/`
2. In your model, use `build_loss` to add your custom loss (see [basic_restorer.py](https://github.com/open-mmlab/mmediting/blob/master/mmedit/models/restorers/basic_restorer.py) as an example.)

**Comment by ACALJJ32** [comment]
And if I want to run a model on my own dataset, how do I implement it?

**Comment by ckkelvinchan** [comment]
Do you mean training or test?

**Comment by ACALJJ32** [comment]
Both training and test

**Comment by ckkelvinchan** [comment]
If you are talking about super-resolution, you can use
- `SRFolderDataset` for image super-resolution, 
- `SRFolderVideoDataset`for video super-resolution (sliding-window framework), and
- `SRFolderMultipleGTDataset` for video super-resolution (recurrent framework).

**Comment by ACALJJ32** [comment]
Thank you.
